Module.yaml

```diff
@@ -0,0 +1,85 @@
+version: 1.0.0
+name: Melita IO for devices controller
+description: Module for communication with melita.io API for devices controller
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_melita_output.png
+type: process
```

Comment:
Last module in the chain so `type: output`

Reply:
updated